### ISSUE TITLE
View Both Raw and Parsed Simultaneously

### ISSUE THREAD

**Issue by Hossy** [callumlocke/json-formatter#178]
I was using this extension recently and came across a pleasant bug: I was able to see the raw and parsed at the same time.  Of course, I can't reproduce it and now it only shows either one or the other.  I think it would be awesome to be able to see the raw (top) and parsed (bottom) in the same view.

**Comment by callumlocke** [comment]
Yeah I've seen that happen before, I think it happens when the browser restarts with a JSON tab open. Just on the first load, it shows the raw JSON at the top. I think it must be something weird with the browser recovering the state of the page and messing it up.

Anyway, I'm curious why you want to be able to see both at the same time?
